anx/provider/loadbalancer/loadbalancer.go

```diff
@@ -285,7 +285,7 @@ func (m mgr) reconciliationForService(ctx context.Context, clusterName string, s
 		externalAddresses = make([]net.IP, 0)
 	}
 
-	mrecon := reconciliation.Multi()
+	mrecon := reconciliation.Multi(m.api, string(svc.UID))
```

Comment:
no changes to multirecon please, instead create a stateRetriever in this place, add it to `ctx` which is then passed to the single recons below